docker-compose-integration.yaml

```
@@ -11,23 +11,6 @@ services:
     links:
```

Additionally, this should `depend_on` the jellyfish to ensure JF has started before running the tests
```suggestion
    depends_on:
      - jellyfish
```